detection_rules/etc/version.lock.json

```diff
@@ -4062,15 +4076,15 @@
       "7.16": {
         "max_allowable_version": 99,
         "rule_name": "AWS CloudTrail Log Created",
-        "sha256": "93ad2845e0f1417bed42d298695e3e9393252efa42ae1e9207da3138ce39f983",
+        "sha256": "452b3ca5d359e3ff768e8c77fc4274ae51aeb2b514fcc589a4bd4f1295f42877",
```

Comment:
Forked rule changed when all rules are min-stacked at 8.3 and thus changes would not backport to this branch.